ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po

```diff
@@ -639,33 +651,34 @@ msgid ""
 "ontario-data-catalogue\">the survey</a> and tell us how we’re doing. </p> "
 "</li> </ul>"
 msgstr "\n"
+"<nav> <div class=\"ontario-page-navigation--full\"> <div class=\"ontario-"
+"page-navigation-content--full\">"
 "<h2 class=\"ontario-small\">\n"
 "  Dans cette page\n"
 "</h2>\n"
-"<ol class=\"table-of-contents\">\n"
-"  <li>\n"
-"    <a href=\"#open-data\">Données ouvertes</a>\n"
+"<ol class=\"ontario-page-navigation-list\">\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#open-data\">Données ouvertes</a>\n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#other-data\">Autres données</a>\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#other-data\">Autres données</a>\n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#licensing\">Licence d’utilisation des données</a>\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#licensing\">Licence d’utilisation des données</a>\n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#developers\">Ressources pour les concepteurs</a>\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#developers\">Ressources pour les concepteurs</a>\n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#training-materials\">Documents de formation</a>\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#training-materials\">Documents de formation</a>\n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#metadata\">Ce que nous vous disons à propos des données (métadonnées)?</a>   \n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#metadata\">Ce que nous vous disons à propos des données (métadonnées)?</a>   \n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#engagement\">Engagement</a>\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#engagement\">Engagement</a>\n"
```

Comment:
You'll need to add the "skip this page navigation" to the french part of the .po file as well. There is currently no translation, so just use the English version for now.

Reply:
The French translation is "Sauter les liens de navigation de cette page"